### ISSUE TITLE
UncommunicativeMethodArgName not always sensible

### ISSUE THREAD

**Issue by ddfreyne** [bbatsov/rubocop#5435]
This isn’t necessary a bug, but I found the behavior unintuitive and want to get a conversation started around what the default whitelist could/should be.

## Expected behavior

`UncommunicativeMethodArgName` does not complain for short variable names that are common:

* `io` (instance of IO)
* `to` (destination )
* `id` (identifier)
* …

## Actual behavior

Common short variable names are accepted.

## Steps to reproduce the problem

The cop fails on the following:

```
def add_edge(from, to)
end
```

```
def remove(id)
end
```

```
def log(level, message, io = $stdout)
end
```

## RuboCop version

4c15154bebcedc62da82d1fa133b1d86fa79abbb from Git (past 0.52.1)

**Comment by bbatsov** [comment]
That's why we made it configurable. :-) Obviously such cops will always yield some false positives, but you just whitelist those (and we can improve the default configuration).

**Comment by bbatsov** [comment]
Take a look [here](https://rubocop.readthedocs.io/en/latest/cops_naming/#naminguncommunicativemethodargname) and tweak the config to your liking.

**Comment by ddfreyne** [comment]
@bbatsov Correct. I’ve updated the ticket to be more clear around what I intend this ticket to be like — coming up with a better default config? (e.g. of all the examples given, `io` makes the most sense to be in there, I believe).

**Comment by bbatsov** [comment]
No argument from me. I'm fine with shipping some default whitelist.

**Comment by garettarrowood** [comment]
I'll add these now.
